content/en/real_user_monitoring/error_tracking/mobile/kotlin-multiplatform.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Kotlin Multiplatform Crash Reporting and Error Tracking
+description: Set up Error Tracking for your Kotlin Multiplatform applications.
+aliases:
+    - /real_user_monitoring/error_tracking/kotlin-multiplatform
+type: multi-code-lang
+code_lang: kotlin-multiplatform
+code_lang_weight: 10
+further_reading:
+- link: '/real_user_monitoring/error_tracking/'
+  tag: 'Documentation'
+  text: 'Get started with Error Tracking'
+- link: '/real_user_monitoring/error_tracking/explorer'
+  tag: 'Documentation'
+  text: 'Visualize Error Tracking data in the Explorer'
+
+---
+
+## Overview
+
+Error Tracking processes errors collected from the Kotlin Multiplatform SDK. 
+
+Enable Kotlin Multiplatform Crash Reporting and Error Tracking to get comprehensive crash reports and error trends. With this feature, you can access:
+
+- Aggregated Kotlin Multiplatform crash dashboards and attributes
+- Deobfuscated Kotlin Multiplatform (iOS and Android) crash reports
+- Trend analysis with Kotlin Multiplatform error tracking
+
+Your crash reports appear in [**Error Tracking**][1].
+
+## Setup
+
+If you have not set up the Kotlin Multiplatform SDK yet, follow the [in-app setup instructions][2] or see the [Kotlin Multiplatform setup documentation][3].
+
+For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
+
+### Android
+
+#### Add NDK crash reporting
+
+Your Android application may be running native code (C/C++) for performance or code reusability reasons. In order to enable NDK crash reporting, use the Datadog NDK library. 
+
+1. Add the Gradle dependency to your Android source set by declaring the library as a dependency in your `build.gradle.kts` file:
+
+```kotlin
+kotlin {
+  sourceSets {
+    androidMain.dependencies {
+      implementation("com.datadoghq:dd-sdk-android-ndk:x.x.x")
+    }
+  }
+}
+```
+
+2. Enable NDK crash collection after initializing the SDK.
+
+``` kotlin
+// in Android source set
+NdkCrashReports.enable()
+```
+
+#### Add ANR reporting
+
+An "Application Not Responding" ([ANR][4]) is an Android-specific type of error that gets triggered when the application is unresponsive for too long.
+
+For any Android version, you can override the default setting for reporting non-fatal ANRs by setting `trackNonFatalAnrs` (available from Android source set only) to `true` or `false` when initializing the SDK.
+
+ANRs are only reported through the RUM (not through Logs) and more information about ANR reporting can be found [here][5].
+
+### iOS
+
```

Comment:
```suggestion

**Note**: Kotlin 2.0.20 or higher is required if crash tracking is enabled on iOS. Otherwise application may hang if crash tracking is enabled due to the compatibility with `PLCrashReporter`. See other dependencies in the [setup][10] instructions.

All uncaught exceptions resulting in a crash will be reported by the Kotlin Multiplatform SDK.

```